js/ui/gallery.js

```diff
@@ -337,6 +338,20 @@ const Gallery = CollectionWidget.inherit({
         this._loadNextPageIfNeeded();
     },
 
+    _createItemByTemplate(itemTemplate, renderArgs) {
+        return itemTemplate.render({
+            model: renderArgs.itemData,
+            container: renderArgs.container,
+            index: renderArgs.index,
+            onRendered: () => {
+                if(!this._resizeCalled) {
+                    this._resizeCalled = true;
+                    this._dimensionChanged();
+                }
+            }
+        });
+    },
+
```

Comment:
Can we add a test? For example, to check that resizeCallback is called after template rendering